docs/data/data-grid/components/components.md

```diff
@@ -212,6 +212,10 @@ This demo below shows how to use the `slotProps` prop and module augmentation to
 
 {{"demo": "CustomFooter.js", "bg": "inline"}}
 
+## 🚧 Using Joy UI components
+
+{{"demo": "JoyUISlots.js", "bg": "inline"}}
```

Comment:
For this PR, can we exclude the demo so that it can be merged? I'd like to keep Joy internal for now.

Reply:
Yes.

Reply:
I moved it to `experiments` page that will not be deployed to production: https://deploy-preview-8067--material-ui-x.netlify.app/x/react-data-grid/experiments/joy-ui/